src/main/resources/cru.properties

```diff
@@ -0,0 +1,12 @@
+# Theme name
```

Comment:
I believe you still need to set a property either globally or on each RegisteredService to specify to use the `cru` theme